### ISSUE TITLE
Carnwenhan not increasing song durations properly

### ISSUE THREAD

**Issue by Flamebuilt** [LegionDark/Issues#425]
**Date & Time**: 10/31/2015 6:00 PM EST
**Frequency**: constant
**FFXI Client Version**: 30150910_3
**Character Name**: Nabri
**Race**: Tarutaru
**Nation**: San d'Oria
**Main Job**: BRD
**Sub Job**: WHM
**Area & Coordinates**: Anywhere
**Party or Solo**: Both
**NPC or Monster Name**: N/A
**ffxiah.com link (for inventory items only)**: http://www.ffxiah.com/item/19089/carnwenhan & http://www.ffxiah.com/item/19719/carnwenhan
**Multi-boxing? (multiple clients on same connection)**: Yes
**Steps To Reproduce The Issue**: With 85 Carnwenhan, songs casted with ONLY Carn equipped were lasting 3 minutes. I upgraded Carnwenhan to 95, songs then only lasted 2min 30s with ONLY Carn equipped. According to BG-Wiki, the 85 should increase song duration by 30%, and the 95 should increase by 40%.

**Comment by TeoTwawki** [comment]
Dev note: `MOD_SONG_DURATION_BONUS = 454;` controls this effect. Change should go into DSP when time allows.

**Comment by Flamebuilt** [comment]
The berserk augment on Coqueror is also not functioning at all, at any level, from what I could tell. Is that something unfixable for now, or should I open another issue for it?

**Comment by TeoTwawki** [comment]
I may be wrong and it got added since, but last I knew there was no implementation at all for that. I actually had to go dig a bit to realize the song duration was added, as most of these mythic effects don't exist in DarkStar yet.

**Comment by Flamebuilt** [comment]
Ah alright. I figured as much. Thanks for looking into the Carn! Was just a little disappointed that upgrading it actually made it worse lol.

**Comment by Kaoriasada** [comment]
Is this something I can fix?  >.>

**Comment by Kaoriasada** [comment]
85 was set at 50 and 95 was set a 25.  Genius coding there.  >.>

I went ahead and fixed ALL the Carnewenhans that we have and corrected the percentages.

The following changes were made:
Carnewenhan (75): 40% -> 10%
Carnewenhan (85): 50% -> 30%
Carnewenhan (95): 25% -> 40%
Carnewenhan (99): 70% -> 50%
Carnewenhan (119): 70% -> 50%
Carnwenhan (119A): 90% -> 50%

This is how it's SUPPOSED to be on retail.  I coded all the mods on every Carnwenhan.  Some of them had extra ones that aren't meant to be there.  Sent to Loki.
